ios/Flutter/Flutter.podspec

```diff
@@ -0,0 +1,18 @@
+#
+# NOTE: This podspec is NOT to be published. It is only used as a local source!
+#
```

Comment:
Any idea what this file is about?

Reply:
Xcode doesn't really have a system for pulling in 3rd-party native code; `pod` is a tool to do it.  The podspec file is used for loading those, just like `gem` in ruby or whatever.  I didn't realize it wasn't supposed to be checked in, I thought they usually were (along with the `.lock` file) but maybe Flutter generates it internally?  Dunno.

Reply:
Oh I'm familiar with podspec files, just wasn't really sure about this specific one... it seems like it's for the framework or something rather than RainbowMonkey? I'm not sure. I'll investigate...